congame-doc/congame-install.scrbl

```diff
@@ -0,0 +1,40 @@
+#lang scribble/manual
+
+@(require "doc-util.rkt")
+
+@title[#:tag "congame-setup"]{Installing and Setting up Congame servers}
+
+Below are step-by-step instructions for installing Congame.
+
+@section{Congame Prerequisites}
+
+@subsection{Preparing PostgreSQL databases}
+
+@section{Installing Congame Packages}
+
+@section{Environment setup}
+
+Copy @filepath{.env.default} to @filepath{.env}. Chief will automatically load the
+variables defined in this file into the environment of the
+subprocesses defined in the @filepath{Procfile} whenever it is run.
+
+The app expects to be run behind an SSL terminated connection (for
```

Comment:
You should not change this value directly in `app.rkt`, you only need to add the flags `CONGAME_WEB_DEBUG=x` and `CONGAME_IDENTITY_DEBUG=x` in the `.env` file and it should do the trick. Directly changing this parameter will cause trouble when pushing to the server (it's fine if it is only used locally).

Reply:
Ah I didn't realize that (updating the parameter was how we did it during my initial setup call). Given that these values are present in the `.env.default` file that ships with the master branch, we should remove this guidance altogether, and add a separate section about what to change for running the server in prod.